RevenueCat/Plugins/iOS/PurchasesUnityHelper.m

```diff
@@ -339,3 +361,35 @@ void _RCCheckTrialOrIntroductoryPriceEligibility(const char *productIdentifiersJ
 
     [_RCUnityHelperShared() checkTrialOrIntroductoryPriceEligibility:productsRequest[@"productIdentifiers"]];
 }
+
+void _RCInvalidatePurchaserInfoCache() {
+    [_RCUnityHelperShared() invalidatePurchaserInfoCache];
+}
+
+void _RCSetAttributes(const char* attributesJSON) {
+    NSError *error = nil;
+    NSDictionary *attributes = [NSJSONSerialization JSONObjectWithData:[convertCString(attributesJSON) dataUsingEncoding:NSUTF8StringEncoding] options:0 error:&error];
```

Comment:
we don't have a strict char count standard yet, but I think we should try to keep lines ideally under 120 where possible

Reply:
one thought for this one would be 
```objective-c
NSData *attributesAsData = [convertCString(attributesJSON) dataUsingEncoding:NSUTF8StringEncoding];
NSDictionary *attributes = [NSJSONSerialization JSONObjectWithData:attributesAsData options:0 error:&error];
```
or even 

```objective-c
NSData *attributesAsData = [convertCString(attributesJSON) dataUsingEncoding:NSUTF8StringEncoding];
NSDictionary *attributes = [NSJSONSerialization JSONObjectWithData:attributesAsData 
                                                           options:0 
                                                             error:&error];
```